metrics/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="mobile.aerogear.org.metrics" />
```

Comment:
We are using a different kind of package (correctly or not) in the other modules `org.aerogear.mobile.[module-name]`